lib/std/coff.zig

```diff
@@ -62,6 +62,21 @@ pub const MachineType = enum(u16) {
     Thumb = 0x1c2,
     /// MIPS little-endian WCE v2
     WCEMIPSV2 = 0x169,
+
+    pub fn toTargetarch(machine_type: MachineType) std.target.Arch {
```

Comment:
I think this a typo, isn't it?
```suggestion
    pub fn toTargetCpuArch(machine_type: MachineType) std.Target.Cpu.Arch {
```